I0328 23:45:54.395903       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21200", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Role/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:45:54.398848       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21200", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ServiceAccount/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:45:54.411801       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21200", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create RoleBinding/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:45:54.416500       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21200", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Deployment/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:45:54.421693       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21200", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Service/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:45:54.426856       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21200", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-pd in  test-cluster successful
I0328 23:45:54.430607       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21200", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-pd-peer in  test-cluster successful
I0328 23:45:54.437388       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21200", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-pd-3731616 for controller TidbCluster/test-cluster successfully
I0328 23:45:54.442373       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21200", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-pd in  test-cluster successful
I0328 23:45:54.452313       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:45:54.452346       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
I0328 23:45:54.554584       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:45:54.554639       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
I0328 23:45:54.646320       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
I0328 23:45:55.545951       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 23:46:03.615984       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: no leader
, service acto-namespace/test-cluster-pd has no endpoints
I0328 23:46:03.642333       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:03.642378       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 23:46:03.703204       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: no leader
, service acto-namespace/test-cluster-pd has no endpoints
E0328 23:46:03.711811       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 23:46:05.612206       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: no leader
, service acto-namespace/test-cluster-pd has no endpoints
E0328 23:46:05.621322       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 23:46:16.268922       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.194.24:2379: connect: connection refused
I0328 23:46:16.297366       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:46:16.297430       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 23:46:16.358757       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.194.24:2379: connect: connection refused
I0328 23:46:16.370486       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21473", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tikv-peer in  test-cluster successful
I0328 23:46:16.374098       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21473", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-tikv-3831336 for controller TidbCluster/test-cluster successfully
I0328 23:46:16.381142       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21473", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-tikv in  test-cluster successful
I0328 23:46:16.397605       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:46:16.397646       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
E0328 23:46:16.460696       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.194.24:2379: connect: connection refused
I0328 23:46:16.508238       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:16.508291       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Get http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores: dial tcp 10.96.194.24:2379: connect: connection refused, requeuing
E0328 23:46:16.576065       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.194.24:2379: connect: connection refused
I0328 23:46:16.600845       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:16.600901       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Get http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores: dial tcp 10.96.194.24:2379: connect: connection refused, requeuing
E0328 23:46:16.756977       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.194.24:2379: connect: connection refused
E0328 23:46:16.767274       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Get http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores: dial tcp 10.96.194.24:2379: connect: connection refused, requeuing
E0328 23:46:19.708902       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.194.24:2379: connect: connection refused
E0328 23:46:19.720061       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Get http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores: dial tcp 10.96.194.24:2379: connect: connection refused, requeuing
E0328 23:46:26.906187       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/cluster,body response: redirect failed
E0328 23:46:26.917521       1 tikv_upgrader.go:249] tikv: no store found for TiKV ordinal 0 of acto-namespace/test-cluster
I0328 23:46:26.941242       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:26.941291       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores?state=2,body response: redirect failed
, requeuing
E0328 23:46:27.007556       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: redirect failed
E0328 23:46:27.020858       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores,body response: redirect failed
, requeuing
E0328 23:46:28.388206       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: redirect failed
I0328 23:46:28.425296       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:28.425373       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores?state=2,body response: redirect failed
, requeuing
E0328 23:46:28.487786       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: redirect failed
I0328 23:46:28.531174       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:28.531232       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores,body response: redirect failed
, requeuing
E0328 23:46:28.587066       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: redirect failed
I0328 23:46:28.642506       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:46:28.642544       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
E0328 23:46:28.783977       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: redirect failed
I0328 23:46:29.250240       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:29.250275       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/config,body response: redirect failed
, requeuing
E0328 23:46:29.392913       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/cluster,body response: redirect failed
I0328 23:46:29.620988       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:29.621043       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores?state=2,body response: redirect failed
, requeuing
E0328 23:46:29.981101       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: redirect failed
I0328 23:46:30.418996       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:30.419051       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores,body response: redirect failed
, requeuing
E0328 23:46:30.600817       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/cluster,body response: redirect failed
E0328 23:46:31.203827       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/config,body response: redirect failed
, requeuing
E0328 23:46:35.617589       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: redirect failed
I0328 23:46:35.651613       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:35.651651       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores,body response: redirect failed
, requeuing
E0328 23:46:35.729092       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/cluster,body response: redirect failed
I0328 23:46:35.776230       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:35.776284       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores,body response: redirect failed
, requeuing
I0328 23:46:35.862320       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21672", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-2(1345819275648764207) is unhealthy
I0328 23:46:35.862420       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21672", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0328 23:46:35.879714       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:35.879769       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 1 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
E0328 23:46:36.010041       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: redirect failed
I0328 23:46:36.036737       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:36.036790       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0328 23:46:36.625337       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/api/v1/cluster,body response: redirect failed
E0328 23:46:36.635427       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
I0328 23:47:05.689135       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21681", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tidb-peer in  test-cluster successful
I0328 23:47:05.702761       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21681", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tidb in  test-cluster successful
I0328 23:47:05.706378       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21681", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-tidb-6662316 for controller TidbCluster/test-cluster successfully
I0328 23:47:05.712106       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"ec3ed432-9ea0-4ad8-9824-c379e36b054f", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"21681", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-tidb in  test-cluster successful
I0328 23:47:06.522866       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:06.642217       1 equality.go:87] Service spec diff for acto-namespace/test-cluster-tidb:   v1.ServiceSpec{
  	Ports: []v1.ServicePort{
  		{
  			... // 3 identical fields
  			Port:       4000,
  			TargetPort: {IntVal: 4000},
- 			NodePort:   0,
+ 			NodePort:   32352,
  		},
  		{
  			... // 3 identical fields
  			Port:       10080,
  			TargetPort: {IntVal: 10080},
- 			NodePort:   0,
+ 			NodePort:   31384,
  		},
  	},
  	Selector:  {"app.kubernetes.io/component": "tidb", "app.kubernetes.io/instance": "test-cluster", "app.kubernetes.io/managed-by": "tidb-operator", "app.kubernetes.io/name": "tidb-cluster"},
  	ClusterIP: "",
  	... // 12 identical fields
  }
I0328 23:47:06.642269       1 tidb_member_manager.go:442] Sync TiDB service acto-namespace/test-cluster-tidb, spec equal: false, annotations equal: true, label equal: true
I0328 23:47:06.894952       1 service_control.go:88] update Service: [acto-namespace/test-cluster-tidb] successfully, kind: , name: test-cluster
I0328 23:47:06.919664       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
E0328 23:47:07.495305       1 pod_control.go:185] failed to update pod acto-namespace/test-cluster-tidb-2 with cluster labels map[app.kubernetes.io/component:tidb app.kubernetes.io/instance:test-cluster app.kubernetes.io/managed-by:tidb-operator app.kubernetes.io/name:tidb-cluster controller-revision-hash:test-cluster-tidb-788bdc6cfc statefulset.kubernetes.io/pod-name:test-cluster-tidb-2 tidb.pingcap.com/cluster-id:7215729656388423727], TidbCluster: test-cluster, err: Operation cannot be fulfilled on pods "test-cluster-tidb-2": the object has been modified; please apply your changes to the latest version and try again
I0328 23:47:07.738357       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:07.895605       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:07.950043       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:08.253541       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:09.454438       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:25.920012       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:25.943895       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:26.266632       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:26.321689       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:27.466625       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:35.698476       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:35.722281       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:36.082480       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully

